src/js/electron/autoUpdater.js

```diff
@@ -1,11 +1,59 @@
 /* @flow */
 import {app, autoUpdater, dialog} from "electron"
 import log from "electron-log"
+import get from "lodash/get"
+import semver from "semver"
+import got from "got"
+import open from "../lib/open"
+
+const getFeedURLForPlatform = (platform) => {
+  return `https://update.electronjs.org/brimsec/brim/${platform}/${app.getVersion()}`
+}
+
+const autoUpdateLinux = async () => {
+  // Check for updates for MacOS and if there are then we assume there is also one for linux
+  const url = getFeedURLForPlatform("darwin-x64")
+  try {
+    const currentVersion = app.getVersion()
+    if (!semver.valid(currentVersion))
+      throw "Invalid current version format: " + currentVersion
```

Comment:
Not a blocker.  When you throw just a string, the stacktrace and file info are not saved in the error. If you want any of those things, then you can `throw new Error("message"). Some debug tooling takes advantace of the extra meta information provided by the Error instance (chrome devtools, jest tests).

Here is a little more detail.
https://stackoverflow.com/a/57077203/3499804

Reply:
Oh this is good to know, thanks!